server/page.js

```diff
@@ -1,10 +1,41 @@
-function page (contents, dehydratedAppState) {
+const serializeJS = require('serialize-javascript')
+
+function page (contents, appState) {
+  var dehydratedAppState = serializeJS(appState)
+
+  function renderMetaTags () {
+    var s = ''
+    var md = appState.archive && appState.archive.metadata ? appState.archive.metadata : null
+    if (!md) return s
+    if (md.title) {
+      s += `<meta name="title" content="${md.title}" />
+            <meta property="og:title" content="${md.title}" />`
+    }
```

Comment:
should we have defaults for title/author/description if the metadata isn't there that describes dat more generally?